pkg/data/values_test.go

```diff
@@ -0,0 +1,170 @@
+package data
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetValue(t *testing.T) {
```

Comment:
I think we might want to have a few more test cases.

Where the function receives: 
1. a slice and multiple keys denoting indices. This will help to be clearer on the doc comment for the func, as I don't know what the behavior we get now.
1. a slice and an empty string as index
1. a map and an empty string as index
1. a slice and no second argument
1. a map and no second argument

Reply:
1. This is covered by [this test](https://github.com/rancher/wrangler/blob/c29289d99068cbd11024b470165423927dcc881c/pkg/data/values_test.go#L96). If you are looking for something different, let me know.

2-5 - I'll take a look at it, those seem reasonable.

Reply:
I've added the tests for 2-5. Let me know if you see anything else here.

Reply:
I think the test you link is slightly different as it works with slices and maps, where some keys are for the map. But a case where there is just a slice and multiple keys denoting indices is different.
The collection would be just a flat slice and the keys are indices. I am curious as to which index is the one chosen for the return value.